### PR TITLE
version 13

### PR DESCRIPTION
**Pull request by yacandy070**
i created new version

### CODE REVIEW

File: demo2.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 public class PrintVersion {
     public static void main(String[] args) {
         // Print "Version 1" to the console
-        System.out.println("Version 12");
+        System.out.println("Version 13");
```
Comment (SharathChandraPN) [inline demo2.java:4]: no need

Comment (SharathChandraPN) [inline demo2.java]: keep the version 12 , no change required.

Comment (yacandy070) [reply demo2.java]: ok as you say.

**Comment by SharathChandraPN** [comment]
grat!

**Comment by SharathChandraPN** [comment]
great!

**Comment by yacandy070** [comment]
thx

**Pull request merged**